framework/error_handlers/error_4xx.py

```diff
@@ -47,6 +47,6 @@ def handle_429(err: HTTPException):
     403: handle_403,
     404: handle_404,
     405: handle_405,
-    418: handle_418,
+    #418: handle_418,
```

Comment:
Any specific reason this was commented out?

Reply:
Quart errored saying it couldn't find `418` in one of its internal things.

Reply:
Most likely not supported cause it was a joke proposal.